src/unitxt/metric_utils.py

```diff
@@ -248,6 +248,13 @@ class MetricResponse(Dataclass):
 # For example, 'http://127.0.0.1:8000/compute'
 UNITXT_REMOTE_METRICS_ENDPOINT = "UNITXT_REMOTE_METRICS_ENDPOINT"
 
+# Classification of the data sent to remote services in terms of its sensitivity.
+# It determines, for each specified metric, what data can be used.
+# For example: '{"metrics.rag.context_relevance": ["public"]}'.
+# This variable should be a dictionary, where each key is a given metric's name,
+# and a value is a list of data classifications used by that metric.
+UNITXT_REMOTE_METRICS_DATA_CLASSIFICATION = "UNITXT_REMOTE_METRICS_DATA_CLASSIFICATION"
```

Comment:
We could call this something like "UNITXT_DATA_CLASSSIFICATION_POLICY"

{"metrics.rag.mt_bench.using.open-ai" : ["public"] , "postprocessor.translate" : ["proprietry"]}